Sketch.java

```diff
@@ -16,21 +16,113 @@ public void settings() {
    * values here i.e background, stroke, fill etc.
    */
   public void setup() {
-    background(210, 255, 173);
+    background(135, 206, 235);
   }
 
   /**
    * Called repeatedly, anything drawn to the screen goes here
    */
   public void draw() {
```

Comment:
missing [level 3 requirement](https://github.com/SACHSTech/ProcessingTask6-CreatingMethods#level-3), "draw a composite object at various locations specified by the method parameters."